src/pymor/core/config.py

```diff
@@ -45,20 +45,20 @@ def _get_dunegdt_version():
     version = 'outdated'
     try:
         version = dune.gdt.__version__
-        if parse(version) < parse('2021.1.2') or parse(version) >= parse('2021.2'):
-            warnings.warn('dune-gdt bindings have been tested for version 2021.1.x (x >= 2) '
+        if parse(version) < parse('2022.1.1') or parse(version) >= parse('2022.2'):
+            warnings.warn('dune-gdt bindings have been tested for version 2022.1.x (x >= 1) '
```

Comment:
Would we not want to simply increase the upper bound? Using `2021.1.2` wheels should still do the yob, or not?

Reply:
Well, the lower bound is no longer tested (as in actively tested in CI), so strictly speaking not warning would be wrong, no?

Somewhat related, [the requirement](https://github.com/pymor/pymor/blob/main/dependencies.py#L59) has no upper bound, while the warning has one. Which makes sense if we do not want to exclude potentially breaking future wheel versions. 

So bottom line, shouldn't the condition for the warning simply be `imported_version != ci_version`, with no ranges?

Reply:
Well, pyMOR unly uses a very small subset of what the wheels provide, so in general we could allow for a range of suitable versions (all compatible ones). But I guess this holds for most deps, and testing this is a nightmare (or rather, it's out of the question to test more than one version of the wheels). Bottom line, we could simply go with warn-if-imported-is-not-ci, and perhaps add something to the warning like YMMW for other versions? (It's not as simple as that though, since our current lower bound is actually a sharp bound as anything below can't work). So in the interest of scalability, I'd also be fine with just your proposal :/

Reply:
> and testing this is a nightmare (or rather, it's out of the question to test more than one version of the wheels)

Actually, I forgot that I have that automated already :)
There's a job that installs all the oldest possible requirements, with the oldest supported python. So changing the check as I proposed would actually lead to a fail in that job.
I guess you're right and it's better to only move the warning's upper bound to whatever is in CI. 
I'll do that and merge then.